organ-utils/descriptions.yaml

```
@@ -4,11 +4,55 @@
 # at least to start, and down the road I fully expect that there will be 
 # changes requested to describe the organ in the HuBMAP context.
 
-Kidney: > # http://purl.obolibrary.org/obo/UBERON_0002113
-  A paired organ of the urinary tract which has the production of urine as its primary function.
-Brain: > # http://purl.obolibrary.org/obo/UBERON_0000955  
-  The brain is the center of the nervous system in all vertebrate, and most invertebrate, animals.
-  Some primitive animals such as jellyfish and starfish have a decentralized nervous system
-  without a brain, while sponges lack any nervous system at all. In vertebrates, the brain is
-  located in the head, protected by the skull and close to the primary sensory apparatus of
-  vision, hearing, balance, taste, and smell[WP].
\ No newline at end of file
+http://purl.obolibrary.org/obo/UBERON_0002097:
+  name: Skin
+  description: The organ covering the body that consists of the dermis and epidermis.
+http://purl.obolibrary.org/obo/UBERON_0000059:
+  name: Large Intestine
+  description: A subdivision of the digestive tract that connects the small
+    intestine to the cloaca or anus. Lacks or has few villi[Kardong].
+http://purl.obolibrary.org/obo/UBERON_0000948:
+  name: Heart
+  description: A myogenic muscular circulatory organ found in the vertebrate
+    cardiovascular system composed of chambers of cardiac muscle. It is the
+    primary circulatory organ.
+http://purl.obolibrary.org/obo/UBERON_0004538:
+  name: Kidney (Left)
+  description: A kidney that is part of a left side of organism.
+http://purl.obolibrary.org/obo/UBERON_0004539:
+  name: Kidney (Right)
+  description: A kidney that is part of a right side of organism.
```

```suggestion
http://purl.obolibrary.org/obo/UBERON_0002113:
  name: Kidney
  description: A paired organ of the urinary tract which has the production of urine as its primary function.
```

- @ngehlenborg -- My understanding of the latest plan is the we just want a single "Kidney" page. Is that correct? Later in this file, there is a single entry for "Lungs".
- @bherr2 -- Your webcomponent does not support generic "Kidney", correct? So the mapping from top-level to subordinate will need to be in a separate ccf.yaml (or whatever you want to call it) that you'll supply: I'll probably reference a raw url at github.

---

We can special case it in our code. Probably something we should handle anyway.